app/assets/stylesheets/views/_transition-landing-page.scss

```diff
@@ -1,6 +1,7 @@
 
 $red: #FF003B;
 $dark-blue: #0055D2;
+$dark-blue-2: #1E1348;
```

Comment:
Can we update the value of `$dark-blue` instead of creating `$dark-blue-2`, it seems like `$dark-blue` is not being used anywhere anymore.